src/pages/Inventory.page.js

```diff
@@ -1,15 +1,77 @@
+import { getItemsListData } from '../HelperFunctions.util';
+
 const { BaseSwagLabPage } = require('./BaseSwagLab.page');
 
 export class InventoryPage extends BaseSwagLabPage {
     url = '/inventory.html';
 
-    get headerTitle() { return this.page.locator('.title'); } //
+    get headerTitle() { return this.page.locator('.title'); } // .header_secondary_container - parent
+
+    get productsSortSelect() { return this.page.locator('.product_sort_container'); }
 
     get inventoryItems() { return this.page.locator('.inventory_item'); }
 
-    get addItemToCartBtns() { return this.page.locator('[id^="add-to-cart"]'); }
+    get inventoryItemName() { return this.page.locator('.inventory_item_name'); }
+
+    get inventoryItemPrice() { return this.page.locator('.inventory_item_price'); }
+
+    get addItemToCartButtons() { return this.page.locator('[id^="add-to-cart"]'); }
 
     async addItemToCartById(id) {
-        await this.addItemToCartBtns.nth(id).click();
+        await this.addItemToCartButtons.nth(id).click();
+    }
+
+    /**
+     *
+     * @param {"za", "az", "lohi", "hilo"} value
+     */
+    async performSorting(value) {
+        await this.productsSortSelect.selectOption(value);
+    }
+
+    async getInventoryItemsList() {
+        return getItemsListData(this.inventoryItems);
+    }
+
+    async getRandomItemsIndexes() {
```

Comment:
Some suggestions:
Move this function to helper
```suggestion
    async getRandomItemsIndexes(length) {
```